independent-projects/qute/core/src/main/java/io/quarkus/qute/TemplateInstance.java

```diff
@@ -178,6 +184,26 @@ default TemplateInstance onRendered(Runnable action) {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Sets the locale.
```

Comment:
Maybe a little bit more detailed description? Something like?

```suggestion
     * Sets the {@code locale} attribute that can be used to localize parts of the template, i.e. to specify the locale for all message bundle expressions in the template.
```